### ISSUE TITLE
Syntex expands to invalid code when module is empty

### ISSUE THREAD

**Issue by sgrif** [serde-rs/syntex#29]
Given the following:

``` rust
// src/lib.rs
include!(concat!(env!("OUT_DIR"), "/lib.rs"));

// src/lib.in.rs
mod foo;
```

Assuming a standard `build.rs`, and that `src/foo.rs` is empty (or contains a macro that expands to nothing for some reason), the resulting code that syntex will generate is `mod foo;` when it should be `mod foo {}` -- otherwise we'll get an error complaining that it can't find `foo.rs` in `OUT_DIR`.

**Comment by erickt** [comment]
@sgrif: Should be fixed in the upcoming v0.27!

**Comment by sgrif** [comment]
<3
